### PR TITLE
Add color provider

### PR DESCRIPTION
**Pull request by mohsen-schibsted**
We are getting the ColorProvider from [Style Dictionary](https://github.com/warp-ds/tokens) too.

And some cleanup

### CODE REVIEW

File: Sources/Colors/BrandsSampleView/BlocketColorsView.swift [hunk 1]
```diff
@@ -2,7 +2,7 @@ import SwiftUI
 
 public struct BlocketColorsView: View {
     public init() {}
-
+
```
Comment (Ali-NaranjiSani) [inline Sources/Colors/BrandsSampleView/BlocketColorsView.swift:5]: There are some white spaces in the following lines that seem unnecessary. These changes aren't needed, right?

File: Sources/Components/Alert/Alert.swift [hunk 2]
```diff
@@ -212,57 +215,57 @@ extension Warp {
 
 private struct ButtonsView: View, Hashable {
     let primaryButtonProvider: Warp.Alert.ButtonConstructor?
-
+
```
Comment (Ali-NaranjiSani) [inline Sources/Components/Alert/Alert.swift:218]: Same white spaces here.

**Review by Ali-NaranjiSani — APPROVED** [review]
Nicely done Mohsen.
The documentation added is excellent!

File: Sources/Colors/BrandsSampleView/BlocketColorsView.swift (hunk 1 above)
Comment (mohsen-schibsted) [reply Sources/Colors/BrandsSampleView/BlocketColorsView.swift:5]: I used `cmd+A` to select all and then `ctrl+I` to reindent to whatever `Xcode` decided is the right indentation 😅
So probably was wrong before

**Pull request merged**